internal/pkg/plan/persist/persist_test.go

```diff
@@ -909,9 +909,12 @@ func (tc *testCase) run(t *testing.T) {
 
 	// Create Fs
 	fs := testhelper.NewMemoryFsFrom(inputDir)
+	envs := env.Empty()
+	envs.Set(`LOCAL_PROJECT_ID`, `12345`)
```

Comment:
`LOCAL_PROJECT_ID` toto tu bude viac krat.
Namiesto toho aby sa ID dodatocne nestavovalo v kode, tak sa nahradi uz pri nacitavani.
V niektorych testoch na ID nezalezalo a musel som to aj tak doplnit.